pymongo/pool.py

```diff
@@ -583,6 +584,7 @@ def _ismaster(self, cluster_time, topology_version,
         awaitable = False
         if performing_handshake:
             self.performed_handshake = True
+            cmd['helloOk'] = True
```

Comment:
The the Spec PR James says we should send helloOk on all isMaster commands, not just the initial handshake: https://github.com/mongodb/specifications/pull/990#issuecomment-870792241

Reply:
Done.